### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by UnchangedOriInten** [hyzhang98/PiNI#3]
Hello,author,may I ask if the loss during training is only the cross entropy loss?

**Comment by HHHHStar** [comment]
Hi, yes, after adding noise to the inputs, the loss during training is calculated using the cross-entropy loss for classification.
